astropy/conftest.py

```diff
@@ -31,3 +31,4 @@
     matplotlib.use('Agg')
 
 PYTEST_HEADER_MODULES['Cython'] = 'cython'
+PYTEST_HEADER_MODULES['astropy_helpers'] = 'astropy_helpers.astropy_helpers'
```

Comment:
I meant to add it to the default one in `astropy.tests.plugins.display` so every package that uses the astropy testing module will have that info printed out (well, in core we don't really need this feature as much). 

Also, it will need to be backported to make any real difference.

Reply:
Oh okay. Well... somehow this patch doesn't work on Travis anyway... Need to investigate more.
```
astropy_helpers: not available
```

Reply:
It is more reliable to do `git submodule status` but that introduces `git` as dependency. :thinking: